tap_crossbeam/client.py

```diff
@@ -53,13 +48,10 @@ def refresh_access_token(self):
 
         self.__access_token = data['access_token']
 
-        self.__user_expires_at = datetime.utcnow() + \
-            timedelta(seconds=data['expires_in'] - 10) # pad by 10 seconds for clock drift
-
     @backoff.on_exception(backoff.expo,
                           (Server5xxError,
                            RateLimitException,
-                           ConnectionError,
+                           requests.exceptions.ConnectionError,
```

Comment:
Is this a pylint thing? It seemed fine to me before

Reply:
This was something I did with Dan involving two different ConnectionError classes. `requests.exceptions.ConnectionError` is different from just `ConnectionError`